tests/common/killring.lisp

```diff
@@ -1,7 +1,8 @@
 (defpackage :lem-tests/killring
   (:use :cl
         :rove
-        :lem/common/killring))
+        :lem/common/killring
+        :lem))
```

Comment:
I would like to make the common package independent of lem.
How about moving the external-option test to tests/killring.lisp and testing in lem/src/killring.lisp?